platform/security/handler/security-handler-oidc/src/main/java/org/codice/ddf/security/handler/oidc/OidcHandlerConfigurationImpl.java

```diff
@@ -179,7 +185,9 @@ OidcClient createOidcClient(
     if (isBlank(callBackUri)) {
       oidcClient.setCallbackUrl(DEFAULT_CALLBACK_URL);
     } else {
-      oidcClient.setCallbackUrl(callBackUri);
+      // Strip additional query parameters from the callBackUri
+      String uri = callBackUri.split("&")[0];
```

Comment:
Issue when using unspecified response_mode with code flow was that the callback uri had extra parameters like state. Stripping that out fixes it.